### DISCUSSION TITLE
Vertical scrolling for the thumbnails ? (axis option)

### DISCUSSION THREAD

**Discussion by 2llco** [fancyapps/ui#353] [Q&A]
Hello,

Thank you for your Fancyapp which is the best of web in my opinion. 

I used Fancybox v3 with the thumbnails to the right and a vertical scrolling (axis option: "y » en V3). Do you plan to add this option in Fancybox V4 ?

Regards

**Comment by nando4** [comment]
I am curious if can alter FancyBox 5 to function in this way too.

**Comment by fancyapps** [comment]
You can find examples for vertical thumbnails here - https://fancyapps.com/showcase/
Sorry, there is currently no option to make them scrollable like in v3.
